components/platform_crypto/shave_trusted_boot/firmware.c

```diff
@@ -14,6 +15,240 @@
 #include "cn_array_utils.h"
 #define memcpy(f,b,s) _memcpy(f,b,s)
 #define memcmp(f,b,s) _memcmp(f,b,s)
+#define free _free
+#define realloc _realloc
+#endif
+
+//#define _Static_assert(...) extern int bogus
+#if !defined(NO_XMSS)
```

Comment:
I am not a big fan of double negatives - lets switch to `USE_XMSS` and then we will have:

* `#define USE_XMSS 1`
* `#if USE_XMSS`